MicrosoftTeams.schema.json

```diff
@@ -450,6 +450,34 @@
                 "type": "string",
                 "maxLength": 2048
             }
+        },
+        "webApplicationInfo": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "AAD application id of the app.",
+                    "maxLength": 64
+                },
+                "resource": {
+                    "type": "string",
+                    "description": "Resource url of app for acquiring auth token for SSO.",
+                    "maxLength": 2048
+                },
+                "scopes": {
```

Comment:
Let's add a description, especially because here "scopes" means something completely different than elsewhere in the manifest.

Reply:
I just removed the scopes for now since they aren't needed yet.